index.js

```diff
@@ -1,2 +1,7 @@
-const s2 = require('bindings')('s2');
+// magic incantation from step 3 @ https://github.com/mapbox/node-pre-gyp#readme
+const binary = require('node-pre-gyp');
+const path = require('path');
+var binding_path = binary.find(path.resolve(path.join(__dirname,'./package.json')));
```

Comment:
trying to understand why we need this now?

Reply:
magic incantation. see the comment.

Reply:
okay, here's an actual answer to both this and the next Q (about tests)

we are dynamically requiring the bindings for our particular platform. This incantation finds that via node-gyp-pre's logic, see below:

Welcome to Node.js v13.6.0.
Type ".help" for more information.
> const binary = require('node-pre-gyp');
undefined
>
> var binding_path = binary.find(path.resolve('./package.json'));
undefined
>
> binding_path
'/Users/blackmad/RadarCode/s2/lib/binding/Release/node-v79-darwin-x64/s2.node'
>